slack_wrapper/api.py

```diff
@@ -0,0 +1,48 @@
+import requests, json
+
+class SlackAPI():
+	def __init__(self, token):
+		self.TOKEN = token
+		self.domain = 'https://slack.com/api/'
+
+	def has_error(self, response_dict, log="An error ocurred:"):
```

Comment:
@raylu I'm planning on adding more functions like `get_channel_list` that will be used for more important API methods and `has_error` will be in each one of them.